### ISSUE TITLE
Wiki: fatal error in code sample in the Coding Standard Tutorial page

### ISSUE THREAD

**Issue by rodrigoprimo** [PHPCSStandards/PHP_CodeSniffer#496]
The Coding Standard Tutorial page contains an example of a sniff to disallow hash comments: https://github.com/PHPCSStandards/PHP_CodeSniffer/wiki/Coding-Standard-Tutorial#disallowhashcommentssniffphp

The code uses curly braces for array offset access, which causes a fatal error in PHP >= 8.0:

```php
if ($tokens[$stackPtr]['content']{0} === '#') {
```

The line above should be replaced with the following:

```php
if ($tokens[$stackPtr]['content'][0] === '#') {
```

**Comment by jrfnl** [comment]
@rodrigoprimo Thanks for reporting this. Fixed now.

**Comment by jrfnl** [comment]
FYI: as I was looking at the tutorial now anyway, I've updated it a little more: https://github.com/PHPCSStandards/PHP_CodeSniffer/wiki/Coding-Standard-Tutorial/_compare/4af8a760d61e175653bf97082d09c7dbe39349c6...10a39f81858b7a8b673773d45d0c3e4f7196f995
